### PR TITLE
adding support for multiple LB

### PR DESCRIPTION
**Pull request by pankagar13**

### CODE REVIEW

File: Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1 [hunk 1]
```diff
@@ -170,40 +170,47 @@ function Get-MachineConnectionInformation
...
 	}
 }
 
 function Get-FrontEndPorts
 {
-	param([string]$backEndPort)
-
-	$portList = @{}
+	param([string]$backEndPort,
+           [System.Collections.Hashtable]$portList)
```
Comment (pankajkhanzode) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: How is postList used? I don't see it getting consumed.

File: Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1 [hunk 2]
```diff
@@ -170,40 +170,47 @@ function Get-MachineConnectionInformation
...
+        Set-Variable -Name winRmHttpsPortMap -Value $winRmHttpsPortMap -Scope "Global"
+        
+        if($lbGroup.Count -gt 0)
+        {
+            foreach($lb in $lbGroup)
+            {
+                $loadBalancer = Get-AzureLoadBalancer -Name $lb.Name -ResourceGroupName $resourceGroupName -ErrorAction Stop -Verbose
+			    Set-Variable -Name loadBalancer -Value $loadBalancer -Scope "Global"
+
+			    $fqdnMap = Get-MachinesFqdnsForLB -resourceGroupName $resourceGroupName
```
Comment (pankajkhanzode) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: have you checked that all code here is re-entrant and safe? and for below methods as well?

**Comment by pankajkhanzode** [comment]
Signed off

File: Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1 (hunk 2 above)
Comment (pankagar13) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: verified with Single vmtemplate and multiple lb both are working.

**Comment by abhinitmodi** [comment]
Signing off.. Make sure you run e2e once before you checkin.

**Pull request merged**